### DISCUSSION TITLE
An annoying popup for an annoying feature

### DISCUSSION THREAD

**Discussion by Glaived** [Christoph142/modern-scroll#181] [General]
https://github.com/Christoph142/modern-scroll/blob/4d098490bdc5592f890278e1ccc634d4fe80245c/Chromium%20(.crx)/background.js#L216-L220

# _hello_again_ feature
Opens a popup asking you what you think about the extension. No matter what your answer is, **it doesn't matter**. So it's already an annoying feature, offering nothing more than to annoy you. Wtf?

https://github.com/Christoph142/modern-scroll/blob/4d098490bdc5592f890278e1ccc634d4fe80245c/Chromium%20(.crx)/options/prefs.js#L337-L355

![image](https://user-images.githubusercontent.com/1835170/148199317-4753d51f-2612-48ac-bc0c-6402b5808b62.png)

Chrome URL: `chrome-extension://ejonaglbdpcfkgbcnidjlnjogfdgbofp/options/options.html#hello_again`

# _thanks_for_using_ feature
A more or less annoying popup. Although it is good to remember that it is possible to support the development of this extension, the fact that there is no option to disable these two popups, for those who would not like to be disturbed, makes this feature unpleasant. https://github.com/Christoph142/modern-scroll/issues/179

![image](https://user-images.githubusercontent.com/1835170/148199511-d192bca2-a39a-4864-ad54-4d70b69f6ffd.png)

Chrome URL: `chrome-extension://ejonaglbdpcfkgbcnidjlnjogfdgbofp/options/options.html#thanks_for_using`

**Comment by Christoph142** [comment]
I think you misunderstood / misread something. Depending on what you select as an answer one of two different dialogs will get shown.
Anyway, I reduced the frequency from 6 weeks to 6 months now in 4.1.1. It's more than 4 times less annoying now.
